### PR TITLE
Tw 1846 update style of image picker

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
- #1846 
- [Figma](https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=27748-127572&node-type=canvas&t=sPZToiwwUHMATNc7-0)
## Need Merge
- https://github.com/linagora/linagora-design-flutter/pull/53
## Impact description
Design update for image picker

## Test recommendations
**Recommendations for how to test this, or anything else you are worried about?**

## Pre-merge
**Does anything else need to be done before merging?**

## Resolved
**Attach screenshots or videos demonstrating the changes**
- Web:
- Android:
image
image

- IOS:

### CODE REVIEW

File: lib/presentation/mixins/media_picker_mixin.dart [hunk 1]
```diff
@@ -96,6 +96,7 @@ mixin MediaPickerMixin on CommonMediaPickerMixin {
     });
 
     return await linagora_image_picker.ImagePicker.showImagesGridBottomSheet(
+      heightOfBottomSheet: SendFileDialogStyle.heightOfBottomSheet,
```
Comment (hoangdat) [inline lib/presentation/mixins/media_picker_mixin.dart:99]: with this, we can not expand the picker when select more. Do you have alternative solution?

Comment (KhaledNjim) [reply lib/presentation/mixins/media_picker_mixin.dart:99]: The picker can be scrolled

**Comment by nqhhdev** [comment]
Screenshot 2024-12-04 at 10 01 48

 I think wrong color for background.
 
 - Background button: `M3/sys light/BG + Surface/background - n99`
 - Background body: `M3/sys light/BG + Surface/surface`

File: lib/presentation/mixins/media_picker_mixin.dart (hunk 1 above)
Comment (nqhhdev) [reply lib/presentation/mixins/media_picker_mixin.dart:99]: Not good for me. Pls refer Telegram app it can be expanded height for picker.

**Comment by nqhhdev** [comment]
Hi @KhaledNjim  any update?

**Comment by hoangdat** [comment]
merged system-design, please update

**Comment by KhaledNjim** [comment]
> merged system-design, please update

Updated

**Comment by KhaledNjim** [comment]
> Screenshot 2024-12-04 at 10 01 48
> I think wrong color for background.
> 
> * Background button: `M3/sys light/BG + Surface/background - n99`
> * Background body: `M3/sys light/BG + Surface/surface`

Updated

image

**Comment by hoangdat** [comment]
> > Screenshot 2024-12-04 at 10 01 48
> > I think wrong color for background.
> > 
> > * Background button: `M3/sys light/BG + Surface/background - n99`
> > * Background body: `M3/sys light/BG + Surface/surface`
> 
> Updated

I didn't see your change for this.

**Comment by KhaledNjim** [comment]
> > > Screenshot 2024-12-04 at 10 01 48
> > > I think wrong color for background.
> > > 
> > > * Background button: `M3/sys light/BG + Surface/background - n99`
> > > * Background body: `M3/sys light/BG + Surface/surface`
> > 
> > 
> > Updated
> 
> I didn't see your change for this.

i updated in commit e04c6ab6452cc799631a9ad1082e59288f4e78a3

**Pull request merged**